### DISCUSSION TITLE
Adding drei to app appears to disable lighting

### DISCUSSION THREAD

**Discussion by amcc** [tommasoturchi/react-three-mind#21] [Q&A]
I have added a videoTexture with drei's useVideoTexture and it works great. This meant installing drei as a dependency.

This appears to stop the lighting working - i'm not sure why. In order to test i did the following:
Took your image-tracking example, installed drei, things still worked initially, but i removed node-modules and package-lock.json and reinstalled and the lighting stopped working.

The reason for the clean install as above is that my full videoTexture version has various warnings and erros that wouldn't go away without a full reinstall. This problem is very likely down to some naievety of mine, i'm fairly new to three/react-three-fiber. Thanks for the amazing work on this already.

**Comment by amcc** [comment]
I made an example to show - interestingly it looks like the built site does have lights, whereas the test site does not. See the example here: https://github.com/amcc/react-three-mind-video
(there's a netlify link to show the built version)

When i run the site from the _start_ script I have a bunch of warnings - these seem relevant to what i'm seeing. I think its due to updates in threejs:

```
WARNING: Multiple instances of Three.js being imported.
```
```
THREE.WebGLRenderer: Property .outputEncoding has been removed. Use .outputColorSpace instead. 
    at Canvas (https://localhost:3000/static/js/bundle.js:31615:5)
    at FiberProvider (https://localhost:3000/static/js/bundle.js:195864:1)
    at CanvasWrapper
    at https://localhost:3000/static/js/bundle.js:176175:17
    at App (https://localhost:3000/static/js/bundle.js:158:80)
```
```
console.js:213 THREE.WebGLRenderer: Property .outputEncoding has been removed. Use .outputColorSpace instead. 
    at Canvas (https://localhost:3000/static/js/bundle.js:31615:5)
    at FiberProvider (https://localhost:3000/static/js/bundle.js:195864:1)
    at CanvasWrapper
    at https://localhost:3000/static/js/bundle.js:176175:17
    at App (https://localhost:3000/static/js/bundle.js:158:80)
```
```
console.js:213 THREE.Texture: Property .encoding has been replaced by .colorSpace. 
    at meshBasicMaterial
    at FallbackMaterial (https://localhost:3000/static/js/bundle.js:136:5)
    at Suspense
    at mesh
    at Plane
    at group
    at wB (https://localhost:3000/static/js/bundle.js:176230:17)
    at https://localhost:3000/static/js/bundle.js:175994:17
    at Suspense
    at Suspense
    at ErrorBoundary (https://localhost:3000/static/js/bundle.js:29291:5)
    at FiberProvider (https://localhost:3000/static/js/bundle.js:195864:1)
    at Provider (https://localhost:3000/static/js/bundle.js:31289:5)
```

here's my app.js:
https://github.com/amcc/react-three-mind-video/blob/main/src/App.js

**Reply by tommasoturchi** [reply]
Ah this I think it's due to different versions used by r3m and your deploy. The dependency management is a bit tricky with all things r3f related. I've updated it yesterday, perhaps try to see if you can align the drei and three version you're using with the ones from the library.
Cheers,

Tommaso
